src/Jackett.Common/Helpers/CookieContainerExtensions.cs

```diff
@@ -40,18 +34,13 @@ public static void FillFromJson(this CookieContainer cookies, Uri uri, JToken js
                     {
                         logger.Info("(Non-critical) Problem loading cookie {0}, {1}, {2}", uri, c, ex.Message);
                     }
-                }
             }
         }
 
-        public static void DumpToJson(this CookieContainer cookies, string uri, JToken json)
-        {
+        public static void DumpToJson(this CookieContainer cookies, string uri, JToken json) =>
```

Comment:
Doing a search for "[braces single line if C#](https://duckduckgo.com/?q=braces+single+line+if+c%23&atb=v193-1&ia=web)" in Duck Duck Go (which doesn't personalize results), I found the following opinions on the first page. I was going to count them and give a report, but there's a ton of opinions in the links.

[This link in particular](https://softwareengineering.stackexchange.com/questions/2715/should-curly-braces-appear-on-their-own-line) shows that the opinions on braces are split fairly even, but what seems to be more important is that whichever style you choose, do not deviate from it. Of the top answers on the thread, almost 300 votes goes to just flip a coin if you can't decide and/or be consistent, and 250 votes goes towards "always use braces"

There's also a handy counter example showing how human error can happen no matter what style you choose about halfway down. No way is perfect, and they all have their flaws. The Microsoft style guides for their documentation and projects in C# all have no braces if all branches are brace free. If one part of an `if - else if -else` block has braces, they all do. Otherwise if none need it, then they don't